src/hotspot/share/jvmci/jvmciCodeInstaller.cpp

```diff
@@ -1178,7 +1178,9 @@ void CodeInstaller::site_Call(CodeBuffer& buffer, u1 tag, jint pc_offset, HotSpo
     CodeInstaller::pd_relocate_JavaMethod(buffer, method, pc_offset, JVMCI_CHECK);
     if (_next_call_type == INVOKESTATIC || _next_call_type == INVOKESPECIAL) {
       // Need a static call stub for transitions from compiled to interpreted.
-      CompiledStaticCall::emit_to_interp_stub(buffer, _instructions->start() + pc_offset);
+      if (CompiledStaticCall::emit_to_interp_stub(buffer, _instructions->start() + pc_offset) == nullptr) {
```

Comment:
We use estimate_stubs_size to presize the buffer to include space for these so it should never fail I think, but we should be checking anyway.  Maybe estimate_stubs_size should be computing space for the aarch64 trampolines?

Reply:
it looks like estimate_stubs_size is computing space for the trampolines via 

`size += trampoline_stubs * CompiledStaticCall::to_trampoline_stub_size();`?

Reply:
It seems like the estimate agrees with the code emitted for the trampoline, so we either have a general problem in this computation or we have some other unexpected use of code cache space.  It isn't critical that we get this estimate right but we do try to avoid resizing during code installation as it can be expensive.

Reply:
Yes, this is a very defensive (and cheap) check that should never fail. Still, better to have it.